src/pages/Users/Users.tsx

```diff
@@ -2,24 +2,48 @@ import { UniversalTable, AddButton } from '@/components'
 import { UsersLoaderData, User } from './types'
 import classes from './Users.module.scss'
 import { useLoaderData } from 'react-router-dom'
+import { ADMIN_ROUTES_NAMES } from '@/routing/routes.names'
+import { Outlet } from 'react-router-dom'
+
+export function mockUsersData() {
```

Comment:
you can already replace this with the real data from firebase

Reply:
![image](https://github.com/user-attachments/assets/56fcb359-636d-4875-85bb-de0199a11061)
done